storage/src/freezer/storage.rs

```diff
@@ -996,8 +995,8 @@ impl<E: Storage + Metrics + Clock, K: Array, V: Codec> Freezer<E, K, V> {
         // Destroy the journal (removes all journal sections)
         self.journal.destroy().await?;
 
-        // Destroy the table
-        self.table.close().await?;
+        // Sync the table before dropping
+        self.table.sync().await?;
```

Comment:
This has the same semantics has before (since `close` called `sync`), but do we really care about syncing here?

Reply:
We can just drop instead